src/Module.php

```diff
@@ -378,6 +383,13 @@ public function renderToolbar($event)
         $view = $event->sender;
         echo $view->renderDynamic('return Yii::$app->getModule("' . $this->id . '")->getToolbarHtml();');
 
+        if (!empty($this->skipAjaxRequestUrl)) {
+            foreach ($this->skipAjaxRequestUrl as $key => $route) {
+                $this->skipAjaxRequestUrl[$key] = Url::to([$route]);
```

Comment:
I don't think array wrapping is needed here

```suggestion
                $this->skipAjaxRequestUrl[$key] = Url::to($route);
```

This way we'll be able to specify either strings or routes including parameters.